adapter/internal/oasparser/envoyconf/access_loggers.go

```diff
@@ -72,7 +72,7 @@ func getFileAccessLogConfigs() *config_access_logv3.AccessLog {
 
 // getAccessLogConfigs provides grpc access log configurations for envoy
 func getGRPCAccessLogConfigs(conf *config.Config) *config_access_logv3.AccessLog {
-	analyticsEnable := conf.Analytics.Enabled
+	analyticsEnable := conf.Analytics.Enabled || conf.Enforcer.Metrics.Enabled
```

Comment:
lets rename the variable grpcAccessLogs enabled

Reply:
Renamed the variable to grpcAccessLogsEnabled in 8d52822bdbad80f19fc2738c59f60af36e75e5c8